tests/config.nims

```diff
@@ -0,0 +1,4 @@
+# this also doesn't work (even in nims): --outdir:"$nimcache/buildTests"
+import os
+let buildDir = currentSourcePath().parentDir.parentDir / "buildTests"
```

Comment:
This seems error prone to me, if I compile the tests from another directory then I'll end up with this `buildTests` directory at some random level of my directory structure.

Reply:
how so?
if this file is in
`/pathto/nimble/tests/config.nims`
buildDir will equal `/pathto/nimble/buildTests`
regardless which dir you compile a test from
`currentSourcePath()` applies to file where it's called (and is absolute).
Try it.